### PR TITLE
Add deployment panel to dashboard

### PR DESCRIPTION
**Pull request by kentakozuka**
Screenshot 2023-06-20 at 20 36 37

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: manifests/pipecd/grafana-dashboards/control-plane/overview.json [hunk 1]
```diff
@@ -343,14 +343,84 @@
...
+          "values": false
+        },
+        "text": {},
+        "textMode": "auto"
+      },
+      "pluginVersion": "8.1.2",
+      "targets": [
+        {
+          "exemplar": true,
+          "expr": "sum(floor(increase(deployment_status{pipecd_component=\"piped\",project=\"$project\",status=~\"DEPLOYMENT_CANCELLED|DEPLOYMENT_SUCCESS|DEPLOYMENT_CANCELLED|DEPLOYMENT_FAILURE\"}[24h])))",
```
Comment (khanhtc1202) [inline manifests/pipecd/grafana-dashboards/control-plane/overview.json:406]: > DEPLOYMENT_CANCELLED|DEPLOYMENT_SUCCESS|DEPLOYMENT_CANCELLED|DEPLOYMENT_FAILURE

There are two `DEPLOYMENT_CANCELLED` in row

Comment (kentakozuka) [reply manifests/pipecd/grafana-dashboards/control-plane/overview.json]: Thanks!

**Comment by kentakozuka** [comment]
@khanhtc1202 Fixed 👍

File: manifests/pipecd/grafana-dashboards/control-plane/overview.json [hunk 2]
```diff
@@ -343,14 +343,84 @@
...
+      "targets": [
+        {
+          "exemplar": true,
+          "expr": "sum(floor(increase(deployment_status{pipecd_component=\"piped\",project=\"$project\",status=~\"DEPLOYMENT_CANCELLED|DEPLOYMENT_SUCCESS|DEPLOYMENT_FAILURE\"}[24h])))",
+          "interval": "",
+          "legendFormat": "",
+          "refId": "A"
+        }
+      ],
+      "title": "Total Deployments in the Last 24h",
```
Comment (khanhtc1202) [inline manifests/pipecd/grafana-dashboards/control-plane/overview.json:412]:
```suggestion
      "title": "Total completed Deployments in the Last 24h",
```
nits

**Comment by khanhtc1202** [comment]
LGTM with small nit 👍

**Comment by kentakozuka** [comment]
@khanhtc1202 Fixed 👍

**Review by khanhtc1202 — APPROVED** [review]
LGTM 👍

**Pull request merged**